### PR TITLE
Clean up i18n in __str__

### PR DESCRIPTION
**Pull request by int-y1**
This affects the following admin pages:
```
Home › Online Judge › Classes › Foo Class in DMOJ: Modern Online Judge
Home › Online Judge › Comments › b:1 by admin
Home › Online Judge › Contest participations › d in Fake Contest 111
Home › Online Judge › Submissions › Submission 12 of testpdf2 by admin
```

`SubmissionSource.__str__` was tested by visiting a submission.

`WebAuthnCredential.__str__` was tested by visiting a user profile.

### CODE REVIEW

File: judge/models/comment.py [hunk 1]
```diff
@@ -166,7 +166,7 @@ def get_absolute_url(self):
         return '%s#comment-%d' % (self.link, self.id)
 
     def __str__(self):
-        return '%(page)s by %(user)s' % {'page': self.page, 'user': self.author.user.username}
+        return _('%(comment_id)s by %(user)s') % {'comment_id': self.page, 'user': self.author.user.username}
```
Comment (quantum5) [inline judge/models/comment.py:169]: Wrong. `self.page` is the page the comment is posted to, not the ID of the comment.

**Review by quantum5 — COMMENTED** [review]
Otherwise LGTM. Note that `__str__` is really only used in the admin.

File: judge/models/comment.py (hunk 1 above)
Comment (int-y1) [reply judge/models/comment.py]: Fixed.

**Review by quantum5 — APPROVED** [review]

**Review by Ninjaclasher — APPROVED** [review]
LGTM

**Pull request merged**